CHANGELOG.md

```diff
@@ -1,10 +1,24 @@
 # Release History
-## 0.9.0 (Unreleased)
+## 0.9.0 (2022-08-31)
 
 ### New Features:
 - Added support for displaying source code as comments in the generated scripts when registering UDFs.
 This feature is turned on by default. To turn if off, pass the new keyword argument `source_code_display` as `False` when calling `register()` or `@udf()`.
+- Added support for `column_names` in `df.write.save_as_table()`. You can now specify the names of columns when appending new data to an existing table.
+- Added function `get_active_session()` in module `snowflake.snowpark.context` to get the current active Snowpark session.
+- Added keyword argument `overwrite` to `session.write_pandas()` to allow overwriting contents of a Snowflake table with that of a Pandas DataFrame.
+- Added `FileOperation.put_stream()` to upload local files to a stage via file stream.
+- Added support for calling table functions from `DataFrame.select()`, `DataFrame.with_column()` and `DataFrame.with_columns` which now take parameters of type `table_function.TableFunctionCall` for columns.
+- Added `TableFunctionCall.alias` to allow aliasing the names of columns that come from the output of table function joins.
 
+### Bug Fixes:
+- Fixed a bug in which batch insert should not raise error when `statement_params` is not passed to the function.
+- Fixed a bug in which column names should be quoted when `create_dataframe` is called with dicts and a given schema.
+- Fxied a bug in which creation of table shall be skipped if the table already exists and is in append mode.
```

Comment:
```suggestion
- Fixed a bug in which creation of table shall be skipped if the table already exists and is in append mode.
```

Reply:
Also, I suggest changing "shall be skipped" to "is skipped".

Reply:
Is this for `copy_into_table`? Better explicitly mention the API

Reply:
updated, from the commit: https://github.com/snowflakedb/snowpark-python/commit/1d61cfab560a1381cb49cdad6693f0ea613dc0a5 it's the `df.write.save_as_table` api.

@sfc-gh-sfan , is ther other api impacted?